application/controllers/UserManagementController.php

```diff
@@ -732,15 +732,17 @@ public function actionRenderUserImport(string $importFormat = 'csv')
                 ['errors' => [gT("You do not have permission to access this page.")], 'noButton' => true]
             );
         }
-
-        $importNote = sprintf(gT("Please make sure that your CSV contains the fields '%s', '%s', '%s', '%s', and '%s'"), '<b>users_name</b>', '<b>full_name</b>', '<b>email</b>', '<b>lang</b>', '<b>password</b>');
-        $allowFileType = ".csv";
-
-        if ($importFormat == 'json') {
-            $importNote = sprintf(gT("Please make sure that your JSON arrays contain the fields '%s', '%s', '%s', '%s', and '%s'"), '<b>users_name</b>', '<b>full_name</b>', '<b>email</b>', '<b>lang</b>', '<b>password</b>');
-            $allowFileType = ".json,application/json";
+        switch ($importFormat) {
```

Comment:
This change should be clarified in the ticket as to have it tested also.

Reply:
> This change should be clarified in the ticket as to have it tested also.

Screenshot of previous behaviour ?

Reply:
Not sure, Just a hint to the tester so he can test it.
If not, changes will be introduced in the app without being reviewed.

Reply:
> Not sure, Just a hint to the tester so he can test it. If not, changes will be introduced in the app without being reviewed.

https://bugs.limesurvey.org/view.php?id=18985#c76266

> To test on 5: same url the document to select must show csv and not (anything)